core/internal/filetransfer/task_ref_art.go

```diff
@@ -131,6 +131,8 @@ func getStorageProvider(ref string, fts *FileTransfers) (ReferenceArtifactFileTr
 		return fts.GCS, nil
 	case uriParts.Scheme == "s3":
 		return fts.S3, nil
+	case uriParts.Scheme == "https" && strings.HasSuffix(uriParts.Host, ".blob.core.windows.net"):
```

Comment:
is it possible to leave a small comment about the naming convention of the URI for azure storage?